### ISSUE TITLE
Compare cell values, if found then update adjacent cell

### ISSUE THREAD

**Issue by milesrs** [dfinke/ImportExcel#900]
Hi,

I'm very new to powershell in general so please bear with me, but hoping someone can help me out!

I'm importing data from somewhere using an API, the data is a simple list of customer names. (this part I've sorted)
I would like to compare these names against an xlsx spreadsheet, if found update the adjacent column (column B) of the customer name found in Column A with a value (for now this is just a static value for testing = 1).
e.g. Column A = list of customer names, Column B = Quantity. 

This is what i have so far below, but after finding the right column and updating he value, nothing happens when i use export-excel.
I know i'm doing something wrong, but after hours of searching this forum and other locations i can't figure it out..

Here's the code:

`
$excel = Open-ExcelPackage -Path 'C:\.xlsx'
$test = Import-Excel -ExcelPackage $excel -WorksheetName Test

$CustomerName = 'test'

    $CustomerName = $test | ? { $_.CustomerName -eq "$CustomerName" }
    $value  = $CustomerName.Quantity = 1
    
$excel | Export-Excel -Path "C:\Users\SOTIwork\Desktop\Monthly Invoicing.xlsx"
Close-ExcelPackage $excel
`

I would be most grateful for any help you can provide me with,

Robert

**Comment by get-name24** [comment]
HI Robert.

I am no expert when using these cmdlets yet, but you don't need to use "Open-ExcelPackage" and "Close-ExcelPackage".  From the videos I've seen Doug explained that those are only used after a file is created and you want to do formatting, like cell boarders, text formatting, etc.  

Also, one thing I noticed is you overwrite your variable $CustomerName.  on your third line you assign it a value of 'test'.  then on the 4th line you assign the variable $CustomerName to the value of your variable "$test", which is your value from line 2.  Since the variable value has changed, you are never searching for your CustomerName of "test"

Just a couple of things I noticed that I hope can help you.
